docs/workloads/azure/backend/java_cqrs/quickstart_java_cqrs.md

```diff
@@ -71,14 +78,44 @@ This is version `1.0.0` of the `java-stacks` project.
     </TabItem>
     <TabItem value="windows">
 
-   ```text
-   mvnw.cmd spring-boot:run
-   ```
+    ```text
```

Comment:
Would be good to add the commands as "bash" not "text". There is no syntax highlight for text. Please refer to this [documentation](https://prismjs.com/#supported-languages)